examples/cisco/demo_hsrp.pp

```diff
@@ -47,6 +56,12 @@
       hsrp_version       => 2,
     }
 
+    cisco_command_config { 'ipv6-addr':
```

Comment:
Why do we need to use cisco_command_config here to add the ipv6 address?  Can't we use cisco_interface for this?